parser/src/main/java/io/specmesh/apiparser/model/Operation.java

```diff
@@ -35,10 +35,12 @@
  * @see <a href=
  *     "https://www.asyncapi.com/docs/reference/specification/v2.4.0#messageTraitObject">messageTraitObject</a>
  */
-@Value
+@Builder
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
```

Comment:
Don't think these are needed for this PR. Not having them reduces the size of your API, which is a good thing.

Also, consider if this API lets the user build specs that are invalid.  This should be avoided. (I think lombok supports _required_ properties, which you should set)

Reply:
changed to this: @AllArgsConstructor(access = AccessLevel.PRIVATE)